README.md

```diff
@@ -1 +1,46 @@
-# Testing
\ No newline at end of file
+# beauty-therapist
+This is a capstone project. It is about a mobile beauty and spa services for women. 
+
+![screenshot](./screenshot.png)
+
+[Live Demo](https://akhtar-joomun.github.io/therapist/)
+
+[Documentation](https://www.loom.com/share/480fb89af5b0497d871cb2f8d53faca5)
```

Comment:
[OPTIONAL]

- Instead of writing documentation, I prefer to write Video presentation. :smiley:

Reply:
I am changing this right now. Thank you for the review. I have amended my project according to the changes you required.